fdbserver/workloads/BlobRestoreWorkload.actor.cpp

```diff
@@ -195,7 +196,12 @@ struct BlobRestoreWorkload : TestWorkload {
 			auto controller = makeReference<BlobRestoreController>(self->extraDb_, normalKeys);
 			state BlobRestorePhase phase = wait(BlobRestoreController::currentPhase(controller));
 			if (phase == BlobRestorePhase::DONE) {
+				// Check if src and dest db are consistent
 				wait(verify(cx, self));
+
+				// Check if we can flush ranges after restore
+				wait(killBlobWorkers(self->extraDb_));
+				wait(flushBlobRanges(self->extraDb_, self, {}));
```

Comment:
i believe also just reading the ranges would be a sufficient check here, although this is good too